rllib/policy/policy.py

```diff
@@ -1567,3 +1595,39 @@ def __repr__(self):
     @Deprecated(new="get_exploration_state", error=True)
     def get_exploration_info(self) -> Dict[str, TensorType]:
         return self.get_exploration_state()
+
+
+def get_gym_space_from_struct_of_tensors(
+    value: Union[Mapping, np.ndarray]
+) -> gym.spaces.Dict:
+    if isinstance(value, Mapping):
+        value_dict = NestedDict(value)
+        struct = tree.map_structure(
+            lambda x: gym.spaces.Box(-1.0, 1.0, shape=x.shape[1:], dtype=x.dtype),
+            value_dict,
+        )
+        space = get_gym_space_from_struct_of_spaces(struct.asdict())
+    elif isinstance(value, np.ndarray):
+        space = gym.spaces.Box(-1.0, 1.0, shape=value.shape[1:], dtype=value.dtype)
```

Comment:
how do you know the limits are -1.0/1.0?

Reply:
Like we discussed offline, this space is only used for sampling values where user have not specified any space for. So its value is not that important. Putting an arbitrary value would do it.